### PR TITLE
update local example doc to link cert-operator statically and other small fixes

### PR DESCRIPTION
**Pull request by fgimenez**
It also includes a change in the service type of vault from ClusterIP to NodePort so that it can be accessed from the host.

### CODE REVIEW

File: examples/local/README.md [hunk 1]
```diff
@@ -51,12 +51,19 @@ the Kubernetes API. For development running Vault in dev mode is fine.
...
 Steps below are optional. It's OK to use a different Vault instance accessible
 from the operator pod. Remember to set `VAULT_HOST` during templates
-preparation accordingly. 
+preparation accordingly.
 
 ```bash
 kubectl apply -f ./vault.yaml
 ```
 
+If you are using minikube you can access vault with:
```
Comment (rossf7) [inline examples/local/README.md]: That's very nice ❤️

File: examples/local/README.md [hunk 2]
```diff
@@ -106,12 +114,12 @@ kubectl get secret -l clusterID=CLUSTER_NAME
...
-Delete the certificate TPOs and the deployment.
+Delete the certificate custom objects and the deployment.
 
 ```bash
-kubectl delete certificate -l clusterID=example-cluster
+kubectl delete certificate -l clusterID=CLUSTER_NAME
 kubectl delete -f ./deployment.yaml
 
-# Optinal. Only when when Vault was set up.
+# Optinal. Only when Vault was set up.
```
Comment (rossf7) [inline examples/local/README.md]: Typo

Comment (fgimenez) [inline examples/local/README.md]: Sure, done

**Comment by fgimenez** [comment]
thank you @rossf7 :)

**Pull request merged**